src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,55 @@
+import classnames from 'classnames';
+import { px } from '../../utils';
+import CloseIcon from '../../assets/close.svg?react';
+import Button from '../Button/Button';
+
+export interface ModalProps extends React.HTMLAttributes<HTMLElement> {
```

Comment:
you can make this more specific:


```suggestion
export interface ModalProps extends React.HTMLAttributes<HTMLDivElement> {
```

Reply:
you might also choose to override the children prop to make it explicit that it's required.